spec.html

```diff
@@ -25810,15 +25822,15 @@ <h1>The AggregateError Constructor</h1>
         </ul>
 
         <emu-clause id="sec-aggregate-error">
-          <h1>AggregateError ( _errors_, _message_ )</h1>
+          <h1>AggregateError ( _errors_, _message_ [ , _options_ ] )</h1>
           <p>When the *AggregateError* function is called with arguments _errors_ and _message_, the following steps are taken:</p>
           <emu-alg>
             1. If NewTarget is *undefined*, let _newTarget_ be the active function object; else let _newTarget_ be NewTarget.
             1. Let _O_ be ? OrdinaryCreateFromConstructor(_newTarget_, *"%AggregateError.prototype%"*, &laquo; [[ErrorData]] &raquo;).
             1. If _message_ is not *undefined*, then
               1. Let _msg_ be ? ToString(_message_).
-              1. Let _msgDesc_ be the PropertyDescriptor { [[Value]]: _msg_, [[Writable]]: *true*, [[Enumerable]]: *false*, [[Configurable]]: *true* }.
-              1. Perform ! DefinePropertyOrThrow(_O_, *"message"*, _msgDesc_).
+              1. Perform ! CreateNonEnumerableDataPropertyOrThrow(_O_, *"message"*, _msg_).
+            1. Perform ? InstallErrorCause(_O_, _options_).
             1. Let _errorsList_ be ? IterableToList(_errors_).
             1. Perform ! DefinePropertyOrThrow(_O_, *"errors"*, PropertyDescriptor { [[Configurable]]: *true*, [[Enumerable]]: *false*, [[Writable]]: *true*, [[Value]]: ! CreateArrayFromList(_errorsList_) }).
```

Comment:
I think lines 1,2,3 should be switched to be 2,3,1. See https://github.com/tc39/proposal-error-cause/issues/37

Reply:
Resolving, per https://github.com/tc39/proposal-error-cause/issues/37#issuecomment-915594650